internal/allocator/allocator_test.go

```diff
@@ -1,13 +1,18 @@
 package allocator
 
 import (
+	"fmt"
 	"math"
 	"net"
 	"strconv"
 	"strings"
 	"testing"
 
 	"go.universe.tf/metallb/internal/config"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
```

Comment:
The imports would be just these (maybe you can use a better name than ptu?)
```suggestion
      "github.com/prometheus/client_golang/prometheus"
      ptu "github.com/prometheus/client_golang/prometheus/testutil"
```

Reply:
Thanks!